### PR TITLE
Porting Inception Model to Tensorflow 1.0 API (tested with RC1)

### PR DESCRIPTION
**Pull request by drelu**
Make Inception (v3) trainable on Tensorflow 1.0RC1.

### CODE REVIEW

File: inception/inception/inception_train.py [hunk 1]
```diff
@@ -132,8 +132,10 @@ def _tower_loss(images, labels, num_classes, scope, reuse_variables=None):
     loss_name = re.sub('%s_[0-9]*/' % inception.TOWER_NAME, '', l.op.name)
     # Name each loss as '(raw)' and name the moving average version of the loss
     # as the original loss name.
-    tf.scalar_summary(loss_name +' (raw)', l)
-    tf.scalar_summary(loss_name, loss_averages.average(l))
+    #tf.scalar_summary(loss_name +' (raw)', l)
+    #tf.scalar_summary(loss_name, loss_averages.average(l))
```
Comment (nealwu) [inline inception/inception/inception_train.py]: Can you remove these two commented lines?

Comment (nealwu) [inline inception/inception/inception_train.py]: Will merge after this.

**Comment by drelu** [comment]
Removed comments.

**Comment by nealwu** [comment]
Thanks! This should be covered by https://github.com/tensorflow/models/pull/1063, so I'm going to close. If there's anything that isn't covered, let me know.

**Pull request closed without merging**